src/RepoAuditor/EntryPoint.py

```diff
@@ -219,14 +220,18 @@ def EntryPoint(
         ),
     ] = [],
     all_warnings_as_error: Annotated[
-        bool, typer.Option("--all-warnings-as-error", help="Treat all warnings as errors.")
+        bool,

```

Comment:
I was surprised to see these changes, as `ruff format` should work in the same way based on settings in `./pyproject.toml`. 

Do you have any idea of why it wanted to change these lines? I'd like to root cause this sooner rather than later, as we may have a bunch of diffs with minor formatting changes over time if we aren't able to determine why those changes have been introduced.